packages/assets-controllers/src/multi-chain-accounts-service/types.ts

```diff
@@ -0,0 +1,27 @@
+export type GetBalancesQueryParams = {
```

Comment:
I'm a lazy dev. I tried generating the types via the OpenAPI spec (we do this for notifications)

```bash
npx openapi-typescript https://accounts.api.cx.metamask.io/docs-json -o ./src/multi-chain-accounts-service/schema.d.ts -t
```

But unfortunately the OpenAPI spec if not fully complete or is loosely typed.

If we own this API, it would be cool to tighten the JSDoc, or API spec to improve type generation.